app/views/employers/employer_profiles/_employer_form.html.erb

```diff
@@ -120,7 +120,8 @@
             <h2> Office Locations </h2>
             <% @row = false %>
             <%= f.fields_for :office_locations, errors: {}, fieldset: false do |office_location,block| %>
-              <%= render partial: 'shared/office_location_fields', locals: {f: office_location} %>
+              <%= render partial: 'shared/office_location_fields', locals: {f: office_location, child_index: 
+              f.object.office_locations.length - 1} %>
```

Comment:
I don't think you want to be specifying child_index like this. It's intended to track the position of a child form element, but if you're setting it to object.office_locations.length - 1 it will be the same for each.

Also, I have a better understanding of the requirements now, so we don't actually need to track every office location form